module/applications/activity/activity-usage-dialog.mjs

```diff
@@ -250,12 +250,16 @@ export default class ActivityUsageDialog extends Dialog5e {
       context.resources = [];
       const isArray = foundry.utils.getType(this.config.consume?.resources) === "Array";
       for ( const [index, target] of this.activity.consumption.targets.entries() ) {
+        const value = (isArray && this.config.consume.resources.includes(index))
+          || (!isArray && (this.config.consume?.resources !== false) && (this.config.consume !== false));
+        const { label, hint, notes, warn } = target.getConsumptionLabels(this.config, value);
+        if ( notes?.length ) context.notes.push(...notes);
```

Comment:
I might have missed some places, but it seemed like the only place we used the newly-added `consumed` parameter was in `consumptionLabelsItemUses`, and it was used to control whether a note was generated. Since we are calling `getConsumptionLabels`, can we assume that any notes returned are related to consumption, so we only need to display them if `consumed` (`value` here) is `true`? That way we don't have to propagate this boolean through the whole `getConsumptionLabels` pipeline. If not then, maybe we pre-emptively make it an `options` object rather than a single boolean.

```suggestion
        if ( notes?.length && value ) context.notes.push(...notes);
```

Reply:
I stuck it into an options object, because I can foresee we may decide to display certain notes even if a certain consumption type is currently unchecked.